paddle/fluid/imperative/layer.h

```diff
@@ -291,7 +291,8 @@ class VarBase {
 
   void ClearGradient() {
     VLOG(1) << "clear gradient of " << Name();
-    if (grads_ && grads_->var_ && grads_->var_->IsInitialized()) {
+    if (grads_ && grads_->IsInitialize() && grads_->var_ &&
```

Comment:
Better to change this in layers.py and throw error when user call it in Variable in Python